packages/loot-core/src/server/budget/template-notes.ts

```
@@ -43,7 +43,7 @@
   categoryWithTemplates.forEach(({ id, name, templates }) => {
     templates.forEach(template => {
       if (template.type === 'error') {
-        errors.push(`${name}: ${template.line}`);
+        errors.push(`${name}: ${template.line}${template.error ? '\nError: ' + template.error : ''}`);
```

ESLint

[error] Replace ``${name}:·${template.line}${template.error·?·'\nError:·'·+·template.error·:·''}`` with `⏎··········`${name}:·${template.line}${template.error·?·'\nError:·'·+·template.error·:·''}`,⏎········`

(prettier/prettier)